src/SDKs/CognitiveServices/CognitiveServices.Tests/CognitiveServices.Tests.csproj

```diff
@@ -27,4 +27,13 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
@felixwa any reason these references are added?
All these references are already implicitly added to your project, you do not need them again.
Please remove, or let me know the reason you made this change?